### PR TITLE
change AHT1X support from initial to full

### PR DESCRIPTION
**Pull request by device111**
- both sensor address where supported
- dual operation support
- timing optimized
- supported commands added

## Description:

**Related issue (if applicable):** fixes #<Tasmota issue number goes here>

## Checklist:
  - [x] The pull request is done against the latest dev branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR.
  - [x] The code change is tested and works on core 2.6.1
  - [x] The code change pass travis tests. **Your PR cannot be merged unless tests pass**
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

### CODE REVIEW

**Comment by arendst** [comment]
Pls revisit the code changes I made to integrate it in the tasmota i2c environment

Fix the use of global temperature and humidity

Fix the use of system provided celsius/fahrenheit

Remove unneeded warnings valid for all i2c drivers when more than one is in the compile set as is the case in tasmota-sensors.

**Comment by arendst** [comment]
Pls revisit the code changes I made to integrate it in the tasmota i2c environment and supporting both global temperature / humidity and celsius/fahrenheit

File: tasmota/xsns_63_aht1x.ino [hunk 1]
```diff
@@ -22,137 +22,160 @@
...
-      WSContentSend_PD(HTTP_SNS_HUM, AHT10.name, humidity);
-#endif // USE_WEBSERVER
+  for (uint8_t i = 0; i < aht1x_count; i++)
+  {
+     float tem,hum;
+     if (AHT1XRead(aht1x_sensors[i].address, tem, hum))
+     {
+       float aht_temp = ConvertTemp(tem);
+       float aht_hum = ConvertPressure(hum);
+
```
Comment (arendst) [inline tasmota/xsns_63_aht1x.ino]: Almost there but ConvertPressure should be ConvertHumidity.

Why don't you do it during AHT1XRead as I did. That way global info is updated every 2 seconds independant of when data is presented.

**Comment by device111** [comment]
uuups...sorry. Shame on me...I've don't have seen the pressure function. I'm blinded...:-)

File: tasmota/xsns_63_aht1x.ino (hunk 1 above)
Comment (device111) [reply tasmota/xsns_63_aht1x.ino]: ok, i rechange code complete to read the sensor every Second and only show the values if there are needed.
Thanks!

**Pull request closed without merging**